hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ScmConfigKeys.java

```diff
@@ -307,6 +307,10 @@
       OZONE_SCM_KEY_VALUE_CONTAINER_DELETION_CHOOSING_POLICY =
       "ozone.scm.keyvalue.container.deletion-choosing.policy";
 
+  public static final String OZONE_SCM_PIPELINE_PER_RAFT_LOG_DISK =
+      "ozone.scm.pipeline.per.raft.log.disk";
```

Comment:
Should be added to `ozone-default.xml`, too (indicated by failure of `TestOzoneConfigurationFields`).